docs/imviz/plugins.rst

```diff
@@ -269,6 +269,23 @@ documented in :ref:`astropy:astropy-table`.
 
 .. _imviz-export-plot:
 
+Catalog Search
+===========
+
+This plugin queries a specified region of the image, marks the sources from the results of the query, and
+provides the number of sources found. After zooming into a specific region of the image, the query uses the center point of the region with a
+radius determined by the farthest edge point of the region. Clicking on :guilabel:`CLEAR` will remove all
+markers on the active viewer.
+
+When you have multiple viewers created in Imviz, the Viewer dropdown menu becomes visible to change the
+active viewer so the query can be based on the image's coordinates.
+
+There is a dropdown menu that allows for selection of a catalog to conduct the source search.
```

Comment:
```suggestion
To select which catalog you would like to use for the search, please pick one of the available options from the catalog dropdown menu. If you have multiple viewers open, you will see another dropdown menu to select the active viewer. The query will be based on the coordinates of the image in the active viewer.
```
Overall looks good! I thought this one section could be worded a little differently and this is my best attempt at that. Feel free to use this suggestion or try something else.

Reply:
Looks good, I'll incorporate that, thank you!